catroidTest/src/org/catrobat/catroid/uitest/ui/fragment/LookFragmentTest.java

```diff
@@ -819,84 +812,52 @@ public void testRenameLookContextMenu() {
 		assertTrue("Look not renamed in actual view", solo.searchText(newLookName));
 	}
 
-	public void testMoveLookUp() {
-		moveLookUp(SECOND_TEST_LOOK_NAME);
-		solo.sleep(TIME_TO_WAIT);
-		solo.goBack();
-		solo.clickOnText(solo.getString(R.string.backgrounds));
-
-		assertEquals("Look didn't move up (testMoveLookUp 1)", SECOND_TEST_LOOK_NAME, getLookName(0));
-		assertEquals("Look didn't move up (testMoveLookUp 2)", FIRST_TEST_LOOK_NAME, getLookName(1));
-	}
-
-	public void testMoveLookDown() {
-		moveLookDown(FIRST_TEST_LOOK_NAME);
-		solo.sleep(TIME_TO_WAIT);
-		solo.goBack();
-		solo.clickOnText(solo.getString(R.string.backgrounds));
-
-		assertEquals("Look didn't move down (testMoveLookDown 1)", SECOND_TEST_LOOK_NAME, getLookName(0));
-		assertEquals("Look didn't move down (testMoveLookDown 2)", FIRST_TEST_LOOK_NAME, getLookName(1));
-	}
-
-	public void testMoveLookToBottom() {
-		moveLookToBottom(FIRST_TEST_LOOK_NAME);
-		solo.sleep(TIME_TO_WAIT);
-		solo.goBack();
-		solo.clickOnText(solo.getString(R.string.backgrounds));
-
-		assertEquals("Look didn't move bottom (testMoveLookToBottom 1)", SECOND_TEST_LOOK_NAME, getLookName(0));
-		assertEquals("Look didn't move bottom (testMoveLookToBottom 2)", FIRST_TEST_LOOK_NAME, getLookName(1));
-	}
+	public void testDragAndDropDown() {
+		for (int i = 0; i < 3; i++) {
+			addLookWithName("TestLook" + i);
+		}
 
-	public void testMoveLookToTop() {
-		moveLookToTop(SECOND_TEST_LOOK_NAME);
-		solo.sleep(TIME_TO_WAIT);
 		solo.goBack();
 		solo.clickOnText(solo.getString(R.string.backgrounds));
 
-		assertEquals("Look didn't move top (testMoveLookToTop 1)", SECOND_TEST_LOOK_NAME, getLookName(0));
-		assertEquals("Look didn't move top (testMoveLookToTop 2)", FIRST_TEST_LOOK_NAME, getLookName(1));
-	}
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(0).getLookName(), FIRST_TEST_LOOK_NAME);
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(1).getLookName(), SECOND_TEST_LOOK_NAME);
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(2).getLookName(), "TestLook0");
```

Comment:
Yes, as you can see, when the test starts, i add some Looks in a for loop. The names of these looks are composed with TestLook plus a unique number (1, 2 and 3 in this case). So the names are composed at runtime. Same goes for the same issue in NFCTagFragmentTest.